backend/python/tests/unit/test_models.py

```diff
@@ -0,0 +1,16 @@
+from app.models.user_mg import User
+
+
+def test_create_user():
```

Comment:
Hmm this test is essentially just testing `mongoengine`'s own `User` function right? Might be more useful to test `UserService` instead, mirroring the TypeScript tests. Could be done in a different PR if it involves a lot additional work :)

Reply:
Yeah that sounds good -- will add in a separate PR